Policy/d8.LayoutBuilderModuleDisabled.policy.yml

```diff
@@ -0,0 +1,21 @@
+title: Layout Builder is not installed
+name: Drupal-8:LayoutBuilderModuleDisabled
+class: \Drutiny\Plugin\Drupal7\Audit\ModuleDisabled
```

Comment:
Should the class be '\Drutiny\Plugin\Drupal8\Audit\ModuleDisabled' ?

See https://drutiny.github.io/2.3.x/audits/DrutinyPluginDrupal8Audit/#moduledisabled